### DISCUSSION TITLE
Enhancement: Settings should be loadable from the environment

### DISCUSSION THREAD

**Discussion by newbeeDotA** [jchristn/Less3#17] [Ideas]
Enhancement request title: Settings should be loadable from the environment
Use case: Load settings from the environment in containerized environment for better synchronization between containers and a easy to deploy application.
Current behavior: Settings cant be loaded form the environment, only from the system.json file.
Requested behavior: Load simple settings (strings, bools, numbers) from the environment.
Recommended implementation: go through all applicable settings and check for a environment var that corresponds with it and overwrite the loaded setting. 
Usefulness of the enhancement: makes containerized applications more easy to deploy without changing the system.json file for each deployment.

PS: Iam already on developing something like this at the moment, so i could do a PR soon on this.

**Comment by jchristn** [comment]
Hi @newbeeDotA thanks, I appreciate the suggestion.  Main reason I haven't done it is it is easy enough to mount a file/folder into the container at startup.  I'd be happy to take a PR on this when you have one ready.  Enjoy the holiday break
